pkg/platform/github.go

```diff
@@ -100,6 +100,17 @@ func NewGitHub(ctx context.Context, cfg *gh.Config) (*GitHub, error) {
 		graphqlClient: graphqlClient,
 	}
 
+	if cfg.GitHubServerURL != "" || cfg.GitHubRunID > 0 || cfg.GitHubRunAttempt > 0 {
+		g.logURL = fmt.Sprintf("%s/%s/%s/actions/runs/%d/attempts/%d", cfg.GitHubServerURL, cfg.GitHubOwner, cfg.GitHubRepo, cfg.GitHubRunID, cfg.GitHubRunAttempt)
+	}
+
+	if cfg.GitHubJobName != "" {
+		resolvedURL, err := g.resolveJobLogsURL(ctx)
+		if err == nil {
```

Comment:
Just want to make sure we are ok with not doing anything about this err?

Reply:
Yes, it's intentional. If there's an error getting the job URL, then it fallbacks to the URL to the entire workflow run (see the assignment directly above).